### ISSUE TITLE
error reply message

### ISSUE THREAD

**Issue by m0hammadhossein** [pytdbot/client#5]
this is my code:

```
@app.on_updateNewMessage()
async def reply_message(c: Client, message: Update):
    print(await message.reply_text('test', quote=True, parse_mode='html'))
```

get this error

```
{
    "@type": "error",
    "code": 400,
    "message": "Failed to parse JSON object as TDLib request: Unknown class \"messageReplyToMessage\""
}
```
version 0.8.6

**Comment by AYMENJD** [comment]
Hey, could you try to install GitHub version:
```bash
pip install git+https://github.com/pytdbot/client.git
```

it should be fixed and later i will release it publicly.

**Comment by m0hammadhossein** [comment]
solved thanks

**Comment by AYMENJD** [comment]
Thank you, this was fixed in Pytdbot v0.8.7 !
